desk/app/activity.hoon

```diff
@@ -135,11 +135,37 @@
   =?  old  ?=(%2 -.old)  (state-2-to-3 old)
   =?  old  ?=(%3 -.old)  (state-3-to-4 old)
   =?  old  ?=(%4 -.old)  (state-4-to-5 old)
-  ?>  ?=(%5 -.old)
+  =?  old  ?=(%5 -.old)  (state-5-to-6 old)
+  ?>  ?=(%6 -.old)
   =.  state  old
-  sync-reads
-  +$  versioned-state  $%(state-5 state-4 state-3 state-2 state-1)
-  +$  state-5  current-state
+  (emit %pass /fix-init-unreads %agent [our.bowl dap.bowl] %poke noun+!>(%fix-init-unreads))
+  +$  versioned-state
+    $%  state-6
+        state-5
+        state-4
+        state-3
+        state-2
+        state-1
+    ==
+  +$  state-6  current-state
+  +$  state-5
+    $:  %5
+        allowed=notifications-allowed:a
+        =indices:a
+        =activity:a
+        =volume-settings:a
+    ==
```

Comment:
Might be able to say something like this. Or too cute? Does make it clear that nothing besides the tag changed.

```suggestion
  +$  state-5  _%*(. *state-6 - %5)
```